### ISSUE TITLE
Livereload no longer seems to reload the browser after rebuilding the app

### ISSUE THREAD

**Issue by franktcurran** [ember-cli/ember-cli#7938]
With the latest version of ember-cli, new and existing apps no longer seem to reload in the browser after a change.
I see that the live reload script has been injected into the HTML, but it's not functioning for whatever reason.
If I install the previous version of ember-cli-inject-live-reload (ember install ember-cli-inject-live-reload@1.7.0), all is well again.
Please let me know if more details would be helpful or if this issue is misplaced.

---

Output from `ember version --verbose && npm --version && yarn --version`:
```
fcurran$ ember version --verbose && npm --version && yarn --version
ember-cli: 3.3.0
http_parser: 2.8.0
node: 6.14.0
v8: 5.1.281.111
uv: 1.16.1
zlib: 1.2.11
ares: 1.10.1-DEV
icu: 58.2
modules: 48
openssl: 1.0.2o
os: darwin x64
3.10.10
0.19.1
```

**Comment by rwjblue** [comment]
Thank you for reporting! I believe this should be fixed by https://github.com/ember-cli/ember-cli-inject-live-reload/pull/50 (released in ember-cli-inject-live-reload@1.8.1). 

Mind confirming?

**Comment by franktcurran** [comment]
@rwjblue, working now, thanks!
